web app/src/main/java/de/tinf15b4/kino/web/ui/SmartCinemaUi.java

```diff
@@ -164,73 +166,105 @@ private void userClicked() {
-        footer.setHeight("100px");
-
-        // First row in the footer
-        HorizontalLayout row1 = new HorizontalLayout();
-        row1.setSpacing(true);
-        row1.setSizeFull();
-
-        Button account = new Button("Mein Konto", e -> navigateTo("account"));
-        row1.addComponent(account);
-        Button contact = new Button("Kontakt", e -> navigateTo("contact"));
-        row1.addComponent(contact);
-
-        row1.setComponentAlignment(account, Alignment.MIDDLE_LEFT);
-        row1.setComponentAlignment(contact, Alignment.MIDDLE_LEFT);
-        row1.setExpandRatio(contact, 1);
-
-        // Second row in the footer
-        HorizontalLayout row2 = new HorizontalLayout();
-        row2.setSpacing(true);
-        row2.setSizeFull();
-
-        Label copyright = new Label("Copyright &copy; 2016 smartCinema", ContentMode.HTML);
-        copyright.addStyleName("copyrightText");
-        copyright.setSizeUndefined();
-        row2.addComponent(copyright);
-        Button impressum = new Button("Impressum", e -> navigateTo("impressum"));
-        row2.addComponent(impressum);
-        Button datenschutz = new Button("Datenschutz", e -> navigateTo("data_security"));
-        row2.addComponent(datenschutz);
-
-        row2.setComponentAlignment(copyright, Alignment.MIDDLE_LEFT);
-        row2.setComponentAlignment(impressum, Alignment.MIDDLE_LEFT);
-        row2.setComponentAlignment(datenschutz, Alignment.MIDDLE_LEFT);
-        row2.setExpandRatio(datenschutz, 1);
-
-        footer.addComponent(row1);
-        footer.addComponent(row2);
-
-        return footer;
-    }
-
     private Component createNavigator() {
         VerticalLayout navigator = new VerticalLayout();
-        navigator.setSpacing(true);
-        navigator.setMargin(true);
         navigator.setSizeUndefined();
+        navigator.setHeight("100%");
+
+        Label movieLabel =  new Label("FILME");
+        movieLabel.setId("navHeadings_Movie");
+        navigator.addComponent(movieLabel);
+
+        Button movieBtn = (Button) (createViewButton("Filme", MovieListView.VIEW_NAME, FontAwesome.PLAY));
+        movieBtn.setId("navigatorBtn_Movie");
+        movieBtn.addStyleName(BaseTheme.BUTTON_LINK);
+        navigator.addComponent(movieBtn);
+
+        Button newMovieBtn = (Button) (createViewButton("Neu im Kino", "new", FontAwesome.EXCLAMATION_CIRCLE));
+        newMovieBtn.setId("navigatorBtn_New");
+        newMovieBtn.addStyleName(BaseTheme.BUTTON_LINK);
+        navigator.addComponent(newMovieBtn);
+
+        Button soonBtn = (Button) (createViewButton("Demnächst", "coming_soon", FontAwesome.HISTORY));
+        soonBtn.setId("navigatorBtn_Soon");
+        soonBtn.addStyleName(BaseTheme.BUTTON_LINK);
+        navigator.addComponent(soonBtn);
+
+        //navigator.addComponent(createViewButton("Neu im Kino", "new", FontAwesome.EXCLAMATION_CIRCLE));
+        // navigator.addComponent(createViewButton("Demnächst", "coming_soon", FontAwesome.HISTORY));
```

Comment:
Why are these commented out? If we don't need these, than we could delete them as well